docker-alfresco/pom.xml

```diff
@@ -200,14 +200,14 @@
                                     <overWrite>false</overWrite>
                                     <outputDirectory>${project.build.directory}/amps</outputDirectory>
                                 </artifactItem>
-                                <artifactItem>
+<!--                                 <artifactItem>
                                     <groupId>org.alfresco.integrations</groupId>
                                     <artifactId>alfresco-googledrive-repo-community</artifactId>
                                     <version>${alfresco.googledrive.version}</version>
                                     <type>amp</type>
                                     <overWrite>false</overWrite>
                                     <outputDirectory>${project.build.directory}/amps</outputDirectory>
-                                </artifactItem>
```

Comment:
FYI: [GoogleDocs 3.2.0-RC3|https://github.com/Alfresco/acs-community-packaging/blob/develop/pom.xml#L99] no longer has a max version so there should be no need to comment-out.